auth0/dto.go

```diff
@@ -79,12 +116,17 @@ const (
 	lastAuth0TokenRequest = "last-auth0-token-request-"
 	auth0TokenCache       = "auth0-token-cache-"
 	tokenDoc              = "token"
+	auth0JwksCache        = "auth0-jwks-cache-"
+	jwksDoc               = "jwks"
 )
 
 // RefreshResult ...
 type RefreshResult string
 
-const(
+// RefreshResultJwks ...
+type RefreshResultJwks string
```

Comment:
Not good to have a type from type unless the new type does receive a new functionalities, reference for that is Duration type which is int64 in origin

Reply:
Addressed!